src/core/extended-language-client.ts

```diff
@@ -18,8 +18,8 @@
  *
  */
 
-import { Selection, Uri } from "vscode";
-import { LanguageClient } from "vscode-languageclient";
+import {Selection, Uri} from "vscode";
+import {LanguageClient} from "vscode-languageclient";
```

Comment:
The space between the curly braces and the imports are a convention we followed throughout the code base. Shall we stick to it?